doc-src/templates/default/fulldoc/html/css/common.css

```diff
@@ -0,0 +1,13 @@
+/* Default color is 888, change to 666 for at least a 4.5 contrast ratio */
+li .item { color: #666; cursor: pointer; }
+#search { width: 170px; position: static; margin: 3px; margin-left: 10px; font-size: 0.9em; color: #666; padding-left: 0; padding-right: 24px; }
+
+/* Override opacity for at least a 4.5 contrast ratio */
+li.collapsed a.toggle {
+    opacity: 1;
+    cursor: default;
```

Comment:
why cursor: default?  What is it otherwise?

Reply:
It's copied from the existing styling, just change opacity.